Pax.cs

```diff
@@ -112,25 +144,16 @@ public class Frontend {
                  true);
       print_kv ("Using configuration file: ", PaxConfig.config_filename);
       print_kv ("Using assembly file: ", PaxConfig.assembly_filename);
-
-      PaxConfig.assembly = Assembly.LoadFile(PaxConfig.assembly_filename);
-
-      var devices = CaptureDeviceList.Instance;
-      Debug.Assert (devices.Count >= 0);
-      if (devices.Count == 0)
-      {
-        Console.ForegroundColor = ConsoleColor.Red;
-        Console.WriteLine("No capture devices found");
-        return -1;
-      }
-
+    }
+    private static void Configure(CaptureDeviceList devices)
```

Comment:
Is there a specific coding convention/style you're using? For consistency you could leave a space between functions.